site/content/en/blog/releases/1.25.0-rc.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "1.25.0 - Release Candidate"
+linkTitle: "1.25.0-rc"
+date: "2022-07-26"
+---
+This is the 1.25.0 release of Agones.
+
+This release has lots of enhancements and bug fixes. This includes:
+
+* Move CustomFasSyncInterval to Beta.
+* Move NodeExternalDNS to stable.
+* Upgrade SuperTuxKart to 1.3.
+* Upgrade Xonotic to 0.8.5.
```

Comment:
```suggestion
* Upgrade Xonotic to 0.8.5.
* Add Alpha SDK support to the Unity SDK
```

Looks good, this was the only addition I would add, since this was a nice addition.